virt/vm_networking/virt-connecting-vm-to-sriov.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: ASSEMBLY
+[id="virt-connecting-vm-to-sriov"]
+= Connecting a virtual machine to an SR-IOV network
+include::_attributes/common-attributes.adoc[]
+:context: virt-connecting-vm-to-sriov
+
+toc::[]
+
+You can connect a virtual machine (VM) to a Single Root I/O Virtualization (SR-IOV) network by performing the following steps:
+
+. xref:../../virt/vm_networking/virt-connecting-vm-to-sriov.adoc#nw-sriov-configuring-device_virt-connecting-vm-to-sriov[Configure an SR-IOV network device.]
```

Comment:
I suggest pulling the bullet out of your link text and starting with a gerund.

Reply:
What does "pulling the bullet out of your link text" mean?

Reply:
> What does "pulling the bullet out of your link text" mean?

@apinnick Sorry was moving a mile a minute. I had meant to say pull the period out of your link text.